lib/src/url_launcher.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:url_launcher/url_launcher.dart';
 
-void openUrl(String url) async {
-  if (await canLaunch(url)) {
-    await launch(url);
+void openUrl(Uri uri) async {
```

Comment:
Maybe it would be easier if we would pass string here and convert it to URI in this function?

We could also do a safecheck:
```
final uri = Uri.tryParse(url);
if(uri != null && canLaunchUrl(uri)) {
  await launchUrl(uri);
}
```